### PR TITLE
Extra error details

### PR DESCRIPTION
**Pull request by ralphtheninja**
Tack on extra information from `.errors` property, if present.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -138,7 +138,7 @@ test('data.message calls back with error', function (t) {
   server = util.makeServer(testData)
     .on('ready', function () {
       ghutils.ghget(xtend(auth), urlBase, {}, function (err, data) {
-        t.deepEqual(err, new Error('Error from GitHub: borked borked'))
+        t.is(err.message, 'Error from GitHub: borked borked')
```
Comment (ralphtheninja) [inline test.js:141]: This is a bug in the tests.

Comment (vweevers) [reply test.js:141]: How so? Does `deepEqual` not compare Error objects?

**Comment by daviwil** [comment]
This will make problems much easier to diagnose, thanks a lot @ralphtheninja!

**Review by vweevers — APPROVED** [review]
LGTM

File: test.js (hunk 1 above)
Comment (ralphtheninja) [reply test.js:141]: I thought it did, but apparently not.

screenshot from 2018-06-22 17-20-48

The `console.log` logs `Error from GitHub: borked borked ([{"foo":"bar"}])` but `t.deepEqual()` says it's ok.

Comment (vweevers) [reply test.js:141]: I see. I'm guessing that's because `message` isn't enumerable.

**Comment by ralphtheninja** [comment]
@vweevers Minor or patch? If minor I have to release a new version of `ghreleases` as well :)

**Comment by vweevers** [comment]
Patch is probably fine because it doesn't break common patterns like `if (/github error/i.test(err))`. But I'm not familiar with this family of modules, your call ;)

If you wanna stay safe you could instead do `Object.defineProperty(err, 'errors', { value: errors })`.

**Pull request merged**